### DISCUSSION TITLE
About the usage of "Instantiate" rather than "PrefabUtility.InstantiatePrefab"

### DISCUSSION THREAD

**Discussion by vitorfgd** [microsoft/MixedRealityToolkit-Unity#10434] [General]
The MRTK Toolbox is an excellent feature and makes my life easier, however, I'm wondering why the default behavior for selecting an item on the list is an Instantiate on the root of the hierarchy. This makes maintainability harder because changing a prefab in the future doesn't reflect on previously created prefabs. 

I was wondering why the `PrefabUtility.InstantiatePrefab` wasn't applied in this case. It's a similar behavior but the prefab is maintained. I was also wondering, if there is a particular reason for using the hierarchy root when it could be contextual, using `GameObjectUtility.SetParentAndAlign(instance, menuCommand.context as GameObject);`

**Comment by keveleigh** [comment]
This is a good question! Offhand, I can't think of a reason we went with `Instantiate` over `PrefabUtility.InstantiatePrefab`, other than maybe familiarity with the former? This is definitely a good improvement suggestion.

I'm going to convert this discussion into an issue, so we can get it logged as an improvement.
